bazel/repositories.bzl

```diff
@@ -821,10 +821,7 @@ def _com_googlesource_chromium_v8():
         name = "com_googlesource_chromium_v8",
         genrule_cmd_file = "@envoy//bazel/external:wee8.genrule_cmd",
         build_file = "@envoy//bazel/external:wee8.BUILD",
-        patches = [
-            "@envoy//bazel/external:wee8.patch",
-            "@envoy//bazel/external:wee8-annobin.patch",
```

Comment:
is the plan to re-enable this patch for annobin in a follow up?

Reply:
Kind of. The plan is to investigate which options are necessary. If we figure out we don't need any of them, we should get rid of this patch permanently.